Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -51,7 +51,7 @@ final class StringExtensionsTests: XCTestCase {
     }
 
     func testFirstCharacterUppercased() {
-        XCTAssertNotNil("hello world".firstCharacterUppercased)
+        XCTAssertNotNil("".firstCharacterUppercased)
```

Comment:
Again, checking that it's not `nil` is always going to be `true`. I meant more like `XCTAssertEqual("".firstCharacterUppercased, "")`

Reply:
Damn lol I totally misunderstood you, my bad.